tailwind.config.js

```diff
@@ -23,6 +23,7 @@ module.exports = {
 				},
 			},
 			boxShadow: {
+				'figma': '0px 4px 4px rgba(0, 0, 0, 0.25)',
```

Comment:
Valami beszédesebb nevet jobb lenne használni, ha már a többinél a használat helye van (események, galéria, etc) akkor itt is jobb lenne pl. partners